dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/event/RumEventMapperTest.kt

```diff
@@ -173,9 +177,70 @@ internal class RumEventMapperTest {
     }
 
     @Test
-    fun `M return null event W map { internal mapper returns NULL }`(forge: Forge) {
+    fun `M not return null ever W map { ViewEvent }`(forge: Forge) {
```

Comment:
How is this test different from the `M map the bundled event W map { ViewEvent }` test above? It performs almost exactly the same thing, and the test implementation doesn't really meet the test name.
1/ We already have a test that ensures that when an event is a ViewEvent, we call this particular mapper, and since it cannot return null (ensured by kotlin), this test here doesn't really add anything. 
2/ If you wanted to, i'd rather say that you should `assertThat(mappedRumEvent).isNotNull()`
3/ More of a discussion, this all relies on a mock that is an Identity Mapper. Do you think it could be worth it trying to have some `mutate` method in Elmyr ? e.g.: `forge.mutate(item)` and returns the same instance with all possible fields mutated ?

Reply:
btw what is the meaning of `M` and `W` in test naming?

Reply:
Totally agree...actually the reason why I added this test was mostly to tell the story about this rule. There is no way we could test this because everything is enforced by the interface. 
Regarding point number 3, I see your point, I am just trying to think if it is worthy as this is going to be on the client side anyway, we only care about what is coming back as an Object and we'll send it further in the pipeline.

Reply:
@0xnm `M` is for `Must` and `W` is for `When`